sql/updates/auth/2015_09_02_00_auth.sql

```diff
@@ -0,0 +1,15 @@
+SET @s = (SELECT IF(
+    (SELECT COUNT(*)
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'realmlist'
+        AND table_schema = DATABASE()
+        AND column_name = 'address6'
+    ) > 0,
+    "SELECT 1",
+    "ALTER TABLE realmlist ADD address6 VARCHAR(255) NOT NULL DEFAULT ''"
+));
+
+PREPARE stmt FROM @s;
+EXECUTE stmt;
+DEALLOCATE PREPARE stmt;
+
```

Comment:
Don't. That re-apply on update is a feature. It does not re-apply under normal conditions (once this is added to the repo).